cloudpub/aws/utils.py

```diff
@@ -141,3 +144,52 @@ def pprint_debug_logging(
     """
     if log.isEnabledFor(logging.DEBUG):
         log.debug("%s\n%s", log_tag, pformat(rsp_log))
+
+
+def is_str_url(url_str: str) -> Optional[Match[str]]:
+    """
+    Check if string is a URL.
+
+    Args:
+        url_str (str)
+            A string represenation of a URL.
+    Returns:
+        Bool
+    """
+    pattern = r"^https:\/\/[0-9A-z.]+.[0-9A-z.]+.[a-z]+$"
```

Comment:
What about using [this suggestion](https://stackoverflow.com/a/7160778) based on [Django's URL validation](https://github.com/django/django/blob/stable/1.3.x/django/core/validators.py#L45)?

Hopefully bot goes happy as well

Reply:
Looks good, I condensed it down a bit to be just http/https and urls. Removing the localhost allowance.

Reply:
tyvm